### PR TITLE
Add a blog of quick recap of China Kubernetes Contributor Summit

### PR DESCRIPTION
**Pull request by windsonsea**
The Kubernetes Contributor Summit in Shanghai is successfully completed. Add a blog here to attract more contributors to join and share.

cc @mrbobbytables @pacoxu

### CODE REVIEW

**Review by pacoxu — APPROVED** [review]
Overall LGTM for the schedule descriptions.

File: content/en/blog/2023/2023-10-15-kcs-shanghai/index.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+the Kubernetes community contributions. Regardless of the passage of time, the community will
+not erase those shining traces. Brilliance can be found in your PRs, issues, or comments.
+It can also be seen in the smiling faces captured in meetup photos or heard through stories passed down among contributors.
+
+## Technical sharing and discussions
+
+Next, there were three technical sharing sessions:
+
+- [sig-multi-cluster](https://github.com/kubernetes/community/blob/master/sig-multicluster/README.md):
+  [Hongcai Ren](https://github.com/RainbowMango), the owner of Karmada, provided an introduction to
```
Comment (jberkus) [inline content/en/blog/2023/2023-10-15-kcs-shanghai/index.md:47]: "owner"?  Maybe "maintainer"?  Or "an owner", either way.

File: content/en/blog/2023/2023-10-15-kcs-shanghai/index.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+multi-cluster management, scheduling, elasticity, AI, and more. For detailed minutes of
+the Unconference meeting, please refer to <https://docs.qq.com/doc/DY3pLWklzQkhjWHNT>.
+
+## China's contributor statistics
+
+The contributor summit took place in Shanghai, with 90% of the attendees being Chinese.
+Within the Cloud Native Computing Foundation (CNCF) ecosystem, contributions from China have been steadily increasing. Currently:
+
+- Chinese contributors account for 9% of the total.
+- Contributions from China make up 11.7% of the overall volume.
```
Comment (jberkus) [inline content/en/blog/2023/2023-10-15-kcs-shanghai/index.md:77]: This probably understates Chinese contributions.  A lot of Chinese contributors use VPNs and may not show up as being from China in the stats.  I don't know a way to get better figures, though.

**Comment by jberkus** [comment]
This looks good except for one minor issue.  Once you get the image link fixed, it should be ready to merge.

File: content/en/blog/2023/2023-10-15-kcs-shanghai/index.md (hunk 2 above)
Comment (windsonsea) [reply content/en/blog/2023/2023-10-15-kcs-shanghai/index.md]: Thanks for you tips. Add a note there. Hope it is helpful.

**Comment by mrbobbytables** [comment]
Thanks! Yeah, getting the image stuff sorted out with Hugo can be a bit annoying -_-

/approve

File: content/en/blog/2023/2023-10-20-kcs-shanghai/index.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+- [helmfile](https://github.com/helmfile/helmfile): [yxxhero](https://github.com/yxxhero)
+  from [GitLab](https://gitlab.cn/) presented how to deploy Kubernetes manifests declaratively,
+  customize configurations, and leverage the latest features of Helm, including Helmfile.
+
+- [sig-scheduling](https://github.com/kubernetes/community/blob/master/sig-scheduling/README.md):
+  [william-wang](https://github.com/william-wang) from Huawei Cloud shared the recent updates and
+  future plans of SIG Scheduling. This SIG is responsible for designing, developing, and testing
+  components related to Pod scheduling.
+
+{{< figure src="/blog/2023/10/20/kcs-shanghai/kcs03.jpeg" alt="A technical session about sig-multi-cluster" caption="A technical session about sig-multi-cluster" >}}
```
Comment (sftim) [inline content/en/blog/2023/2023-10-20-kcs-shanghai/index.md:61]:
```suggestion
{{< figure src="/blog/2023/10/20/kcs-shanghai/kcs03.jpeg" alt="A group of contributors sat at tables, listening to a presenter speaking at a podium" caption="A technical session about SIG Multicluster" >}}
```

File: content/en/blog/2023/2023-10-20-kcs-shanghai/index.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+slug: kcs-shanghai
+date: 2023-10-20
+author: "Paco Xu and Michael Yao (DaoCloud)"
+---
+
+On September 26, 2023, the first day of
+[KubeCon + CloudNativeCon + Open Source Summit China 2023](https://www.lfasiallc.com/kubecon-cloudnativecon-open-source-summit-china/),
+nearly 50 contributors gathered in Shanghai for the Kubernetes Contributor Summit.
+
+{{< figure src="/blog/2023/10/20/kcs-shanghai/kcs04.jpeg" alt="All participants in the 2023 Kubernetes Contributor Summit" caption="All participants in the 2023 Kubernetes Contributor Summit" >}}
```
Comment (sftim) [inline content/en/blog/2023/2023-10-20-kcs-shanghai/index.md:13]:
```suggestion
{{< figure src="/blog/2023/10/20/kcs-shanghai/kcs04.jpeg" alt="Kubernetes contributors posing for a group photo" caption="All participants in the 2023 Kubernetes Contributor Summit" >}}
```

File: content/en/blog/2023/2023-10-20-kcs-shanghai/index.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+Following the opening remarks, the contributors introduced themselves briefly. Most attendees were from China,
+while some contributors had made the journey from Europe and the United States specifically for the conference.
+Technical experts from companies such as Microsoft, Intel, Huawei, as well as emerging forces like DaoCloud,
+were present. Laughter and cheerful voices filled the room, regardless of whether English was spoken with
+European or American accents or if conversations were carried out in authentic Chinese language. This created
+an atmosphere of comfort, joy, respect, and anticipation. Past contributions brought everyone closer, and
+mutual recognition and accomplishments made this offline gathering possible.
+
+{{< figure src="/blog/2023/10/20/kcs-shanghai/kcs06.jpeg" alt="Face to face meeting in Shanghai" caption="Face to face meeting in Shanghai" >}}
```
Comment (sftim) [inline content/en/blog/2023/2023-10-20-kcs-shanghai/index.md:30]:
```suggestion
{{< figure src="/blog/2023/10/20/kcs-shanghai/kcs06.jpeg" alt="A group of Kubernetes contributors sat around circular tables" caption="Face to face meeting in Shanghai" >}}
```

**Review by sftim — COMMENTED** [review]
Hi.

Post-merge nits about accessibility. `alt` attributes should describe what is seen visually, rather than the context for the photo. Captions (can) provide context.

**Pull request merged**